src/root/admin-contributions.js

```diff
@@ -138,7 +138,7 @@ const adminContributions = {
     },
 
     view(ctrl) {
-        return [
+        return m('', [
```

Comment:
When we need a wrapper, we can add at least an ID or something to help identify the node inside the DOM.